docs/UsersGuide/best/documentation.md

```diff
@@ -66,10 +66,10 @@ change these settings.
 | PROJECT_BRIEF  | Brief description of project |
 | PROJECT_LOGO | 200x50 or smaller logo for project. Leave blank for no logo |
 
-## Github Pages Notes
+## GitHub Pages Notes
```

Comment:
This is the brand, and it's a relatively harmless change (unlike above). Technically this is an anchor, so someone linking to #Github-Pages-Notes will probably break, but, it's a fairly harmless break.